docs/reference/index-modules/index-sorting.asciidoc

```diff
@@ -0,0 +1,105 @@
+[[index-modules-index-sorting]]
+== Index Sorting
+
+When creating a new index in elasticsearch it is possible to configure how the Segments
+inside each Shard will be sorted. By default Lucene does not apply any sort and uses the
+internal _doc_id_ to do the ordering.
```

Comment:
I think saying that segments are ordered by doc id is a bit confusing, it rather works the other way: the ordering of documents inside a segment defines doc ids? Maybe just keep it to a minimum, eg. `By default Lucene does not apply any sort.`.